docs/articles/runmetrics.md

```diff
@@ -11,6 +11,24 @@ weight = 4
 
 # Runtime metrics
 
+## Docker stats
```

Comment:
Thanks @ankushagarwal for the tweak.  Some adjustments, active voice, case and I think  "running" is implied by "runtime" and "currently" is not needed as "supports" says it all.

--> https://gist.github.com/moxiegirl/59ee4f1d32d9aec54a0b

You can use the `docker stats` commadn to live stream a container's runtime metrics. The command supports CPU, memory usage,
memory limit, and network IO metrics.